### PR TITLE
Update 01-introduction.md

### PR DESCRIPTION
**Pull request by orchid00**
little fixes - Introduction.
typos and added links

### CODE REVIEW

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -13,9 +13,10 @@ keypoints:
...
+Scientific research nowadays relies heavily on the computational aspects provided by computer software, yet software is not always developed following practices that ensure its quality and sustainability. A publication that addresses this is the ([Four simple recommendations to encourage best practices in research software](https://f1000research.com/articles/6-876/v1)) provided a simple, yet robust framework of simple recommendations that encourage the adoption of existing best practices in developing research software. These recommendations are designed around Open Science values, and provide practical suggestions that contribute to making research software and its source code more discoverable, reusable and transparent.
```
Comment (fpsom) [inline _episodes/01-introduction.md]: "A publication that addresses this is the [] provided a" -> "A publication that addresses this is the [] **which** provide**s** a"

File: _episodes/01-introduction.md [hunk 2]
```diff
@@ -13,9 +13,10 @@ keypoints:
...
+
+Based on these recommendations, this lesson focuses on providing both the underlying context, as well as, some practical exercises towards establishing their usefulness in the long term. The following episodes of this lesson are structured in the form of one episode per recommendation:
```
Comment (fpsom) [inline _episodes/01-introduction.md]: ", as well as," -> remove the commas

File: _episodes/01-introduction.md [hunk 3]
```diff
@@ -25,7 +26,7 @@ Based on these recommendations, this lesson focuses on providing both the underl
...
+Discussing best practices to develope research software, is bound to touch on the subject of Open Science. Modern research relies on software, and building upon or reproducing that research requires access to the full source code behind that software ([ref](https://open-science-training-handbook.gitbook.io)). Sharing software used for research (whether computational in nature, or that relies on any software-based analysis/interpretation) is a necessary, though not sufficient, condition for reproducibility. In addition to reproducibility, sharing software openly allows developers/scientists to receive career credit for their efforts, either through direct citation or via published software articles. We are going to be discussing all of these aspects in the following lesson.
```
Comment (fpsom) [inline _episodes/01-introduction.md]: "develope research software, is bound" -> remove the comma, typo in develop

"(whether computational in nature, or that relies on any software-based analysis/interpretation)" -> "(whether computational in nature, or relying on any software-based analysis/interpretation)"

**Review by fpsom — CHANGES_REQUESTED** [review]
Thanks @orchid00 ! I've added some minor edits that need to be fixed. Other than these, this LGTM! :)

**Review by orchid00 — COMMENTED** [review]
@fpsom I've addressed the changes, not exactly sure if the PR is updated

**Review by fpsom — APPROVED** [review]

**Comment by fpsom** [comment]
Thanks @orchid00 ! Merging and closing!

**Pull request merged**